F5 Networks/f5-big-ip/ingest/parser.yml

```
@@ -0,0 +1,227 @@
+          PAYLOAD_NEW_SESSION: 'New session from \s*client IP %{IP:src_ip} \(%{DATA}\) at VIP %{IP:dest_ip}.*'
+
+  - name: set_apm_event
+    filter: "{{ parse_apm_event.message != None }}"
+
+  - name: set_cef_fields
+    filter: "{{ parse_cef_event.message != None }}"
+
+  - name: set_grokked_fields
+    filter: "{{ parse_grok_event.message != None }}"
+
+  - name: set_kv_fields
+    filter: "{{ parse_kv_event.message != None }}"
+
+stages:
+  set_common_fields:
+    actions:
+      - set:
+          os.family: "linux"
+          os.platform: "linux"
+
+  set_grokked_fields:
+    actions:
+      - set:
+          rule.name: "{{parse_grok_event.message.rule_name}}"
+
+          event.reason: "{{parse_grok_event.message.msg}}"
+          user.name: "{{parse_grok_event.message.user_name}}"
+          action.name: "{{parse_grok_event.message.action_name}}"
+          process.command_line: "{{parse_grok_event.message.process_command_line}}"
+
+          action.type: "{{parse_grok_event.message.action_type}}"
+          action.outcome: "{{parse_grok_event.message.action_outcome}}"
+          network.transport: "{{parse_grok_event.message.network_transport.lower()}}"
+          source.ip: "{{parse_grok_event.message.source_ip}}"
+          source.port: "{{parse_grok_event.message.source_port}}"
+          destination.ip: "{{parse_grok_event.message.destination_ip}}"
+          destination.port: "{{parse_grok_event.message.destination_port}}"
+
+          url.original: "{{parse_grok_event.message.url_original}}"
+
+          url.full: >
+            {%- if parse_grok_event.message.url_original.startswith('http') -%}
+              {{parse_grok_event.message.url_original}}
+            {%- elif parse_grok_event.message.url_original.endswith(':443') -%}
+              https://{{parse_grok_event.message.url_original.strip(':443')}}
+            {%- elif parse_grok_event.message.url_original.endswith(':80') -%}
+              http://{{parse_grok_event.message.url_original.strip(':80')}}
+            {%- endif -%}
+
+          http.response.status_code: "{{parse_grok_event.message.http_response_status_code}}"
+          tls.version_protocol: "{{parse_grok_event.message.tls_version_protocol.lower()}}"
+          tls.version: "{{parse_grok_event.message.tls_version.lower()}}"
+
+      - set:
+          action.target: "network-traffic"
+        filter: "{{parse_grok_event.message.source_ip != None and parse_grok_event.message.destination_ip != None and parse_grok_event.message.action_type != None}}"
+
+  set_cef_fields:
+    actions:
+      - set:
+          action.target: "network-traffic"
+        filter: "{{parse_cef_event.message.src != None and parse_cef_event.message.dst != None and parse_cef_event.message.act != None}}"
+
+      - set:
+          "@timestamp": "{{parse_timestamp.datetime}}"
+          event.action: "{{parse_cef_event.message.act}}"
```

For backward compatibility:
```suggestion
          event.action: "{{parse_cef_event.message.act}}"
          action.name: "{{parse_cef_event.message.act}}"
```